.github/actions/build-reportstream-functions/action.yml

```diff
@@ -7,6 +7,9 @@ inputs:
 runs:
   using: composite
   steps:
+    - uses: actions/setup-node@v4
```

Comment:
needed to add this step to specify the node version otherwise it was running on node 18 and erroring